Gemfile

```diff
@@ -5,4 +5,5 @@ gemspec
 group :test do
   gem "codeclimate-test-reporter", require: false
   gem "rspec", "~> 3.1"
+  gem "json", "< 2.0.0" if RUBY_VERSION < '2'
```

Comment:
now it not neccessary https://github.com/flori/json/commit/3015920d6eccc2885f5a9ac2d5531800c4bebc24